File: python/sprint5/D/code.py
# ! change LOCAL to False before submitting !
# set LOCAL to True for local testing

LOCAL = True

if LOCAL:
    class Node:  
        def __init__(self, value, left=None, right=None):  
            self.value = value  
            self.right = right  
            self.left = left

    
def solution(root1, root2) -> bool:
    #  Your code
    #  “ヽ(´▽｀)ノ”
    pass


def test():
    node1 = Node(1,  None,  None)
    node2 = Node(2,  None,  None)
    node3 = Node(3,  node1,  node2)

    node4 = Node(1, None,  None)
    node5 = Node(2, None, None)
    node6 = Node(3, node4, node5)
    
    assert solution(node3, node6)

if __name__ == '__main__':
    test()

File: js/sprint5/D/code.js
if (process.env.REMOTE_JUDGE !== 'true') {
    class CNode {
        constructor(value, left = null, right = null) {
            this.value = value;
            this.left = left;
            this.right = right;
        }
    }
}

function solution(root1, root2) {
    // Your code
    // “ヽ(´▽｀)ノ”
}

function test() {
    var node1 = new CNode(1,  null,  null);
    var node2 = new CNode(2,  null,  null);
    var node3 = new CNode(3,  node1,  node2);

    var node4 = new CNode(1,  null,  null);
    var node5 = new CNode(2,  null,  null);
    var node6 = new CNode(3,  node4,  node5);
    
    console.assert(solution(node3, node6));
}